src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/TypedParseJSON.txt

```diff
@@ -0,0 +1,97 @@
+#SETUP: AllowTypeLiteral
+
+// Primitives
+>> ParseJSON("5", Number)
+5
+
+>> ParseJSON("98765.4321", Number)
+98765.4321
+
+>> ParseJSON("""HelloWorld""", Type(Text))
```

Comment:
>Type(Text) [](http://example.com/codeflow?start=31&length=10)

Why do we need to say `Type(Text)` and not just `Text`? Would both work?

Reply:
both would work , will add both